src/pages/Home/index.js

```diff
@@ -0,0 +1,534 @@
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+              <Grid item xs={12} ms={6} md={4}>
+                <Card className={classes.card}>
+                  <CardMedia
+                    component="img"
+                    src={require("./assets/difficulty-english.svg")}
+                    alt="green iguana"
+                  />
+                  <CardContent className={classes.cardConten}>
+                    <Typography align="center">
+                      I face difficulty in understanding and speaking English
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+              <Grid item xs={12} ms={6} md={4}>
+                <Stack alignItems="center">
+                  <img src={require("./assets/down-swirly.svg")} />
+                </Stack>
+                <Card>
+                  <CardContent className={classes.cardConten}>
+                    <Typography align="center">
+                      Learn through interactive classes and self study material
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+              <Grid item xs={12} ms={6} md={4}>
+                <Stack alignItems="center">
+                  <img src={require("./assets/down-swirly.svg")} />
+                </Stack>
+                <Card>
+                  <CardContent className={classes.cardConten}>
+                    <Typography align="center">
+                      Get accurate and fast with our typing guru track
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+              <Grid item xs={12} ms={6} md={4}>
+                <Stack alignItems="center">
+                  <img src={require("./assets/down-swirly.svg")} />
+                </Stack>
+                <Card>
+                  <CardContent className={classes.cardConten}>
+                    <Typography align="center" gutterBottom>
+                      Become confident with our spoken English track
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+            </Grid>
+          </Container>
+          {/* ********************** Section 5 ******************* */}
+          <Container maxWidth="sm">
+            <Stack alignItems="center">
+              <img src={require("./assets/down-swirly.svg")} />
+            </Stack>
+            <Typography sx={{ mt: 3 }} align="center" gutterBottom>
+              Our aim is to help you get technical and soft skills to become
+              ready for advanced learning schools such as Zoho or Navgurukul
+            </Typography>
+          </Container>
+
+          {/* ********************** Section 6 ******************* */}
+
+          <Container maxWidth="md" sx={{ mt: 10 }}>
```

Comment:
As with functions, you'll generally want to keep Components small and single-task specific so you can potentially reuse them and test/modify them more easily.  As an example, you may want to create a `VolunteerSection` component that takes an Array of volunteer names as a property.  You could then later change this to say a carousel or easily update the list of volunteers you display by simply modifying the Array.

Reply:
yes sure